Resources/Textures/_DV/Objects/Consumable/Drinks/fiveoclock.rsi/meta.json

```
@@ -5,7 +5,7 @@
         "y": 32
     },
     "license": "CC-BY-SA-3.0",
-    "copyright": "https://github.com/discordia-space/CEV-Eris/raw/f7aa28fd4b4d0386c3393d829681ebca526f1d2d/icons/obj/drinks.dmi. Edited and fill levels added by Verybigman311.",
+    "copyright": "Made by Verybigman311.",
```

I would still leave this attribution in there. Even if you re-made it, I would put where the original sprite came from, as they are very similar still.


```suggestion
    "copyright": "Original sprite from https://github.com/discordia-space/CEV-Eris/raw/f7aa28fd4b4d0386c3393d829681ebca526f1d2d/icons/obj/drinks.dmi. Remade by Verybigman311.",
```

---

alright